Makefile

```diff
@@ -16,6 +16,9 @@ check: all
 	  echo 'Compiled files were updated, but should have been included/committed.'; \
 	  exit 1; }
 
+checkqa: all
```

Comment:
Quality Assurance.

`checkpy` is also fine and more explicit - I've just thought that it could later also lint JS (but that can be done via `checkjs` then.

Reply:
hmm, I think `checkpy` and `checkjs` is straightforward and simple, especially for japanese people... 😎

Reply:
Oh, missed that.  Will do it.